src/components/inputs/TextInput.js

```diff
@@ -381,13 +397,13 @@ export default class TextInput extends BaseInput {
   }
 
   render() {
-    const {expandable, containerStyle} = this.props;
-    const underlineStyle = this.getUnderlineStyle();
+    const {expandable, containerStyle, underlineColor} = this.props;
+    const underlineStyle = this.getStateColor(underlineColor, true);
```

Comment:
now that it returns color and not a style, let's name the const - underlineColor